packages/libuserspace_rcu/libuserspace_rcu.spec

```diff
@@ -0,0 +1,67 @@
+Name: %{_cross_os}userspace-rcu
+Version: 0.14.0
+Release: 1%{?dist}
+Summary: RCU (read-copy-update) implementation in user-space
+License: LGPL-2.1-or-later
+URL: http://liburcu.org
+Source0: http://lttng.org/files/urcu/userspace-rcu-latest-0.14.tar.bz2
+
+BuildRequires: %{_cross_os}glibc-devel
+
+%description
+%{summary}.
+
+%package devel
+Summary: Development package for %{name}
+Requires: %{name}
+
+%description devel
+%{summary}.
+
+%prep
+%autosetup -n userspace-rcu-%{version} 
+
+%build
+%cross_configure 
+
+# "fix" rpath
+sed -i 's|^hardcode_libdir_flag_spec=.*|hardcode_libdir_flag_spec=""|g' libtool
+sed -i 's|^runpath_var=LD_RUN_PATH|runpath_var=DIE_RPATH_DIE|g' libtool
+
+%make_build
+
+%install
+%make_install
+
+find %{buildroot} -type f -name "*.a" -delete
+
+%files
+%license LICENSE
+%{_cross_attribution_file}
+	
+%{_cross_libdir}/liburcu-bp.so.8*
+%{_cross_libdir}/liburcu-cds.so.8*
+%{_cross_libdir}/liburcu-common.so.8*
+%{_cross_libdir}/liburcu-mb.so.8*
+%{_cross_libdir}/liburcu-memb.so.8*
+%{_cross_libdir}/liburcu.so.8*
```

Comment:
Can we just include this version of the library, or whichever one xfsprogs uses? I don't want to ship a bunch of different implementations when we only use one.

Reply:
I was able to determine xfsprogs only links against `liburcu.so.8` and `liburcu-common.so.8`